dashboard/app/graphs.go

```diff
@@ -187,6 +187,44 @@ func handleFoundBugsGraph(c context.Context, w http.ResponseWriter, r *http.Requ
 	return serveTemplate(w, "graph_histogram.html", data)
 }
 
+func handleCoverageGraph(c context.Context, w http.ResponseWriter, r *http.Request) error {
+	hdr, err := commonHeader(c, r, w, "")
+	if err != nil {
+		return err
+	}
+	hist, err := MergedCoverage(c, "upstream", time.Now().Add(-24*time.Hour*32), time.Now())
+	if err != nil {
+		return err
+	}
+	today := time.Now()
+	dates := []string{}
+	for i := 31; i > 0; i-- {
+		dates = append(dates, today.Add(-1*24*time.Hour*time.Duration(i)).Format(time.DateOnly))
+	}
+	cols := []uiGraphColumn{}
+	for _, date := range dates {
+		if _, ok := hist.covered[date]; ok {
+			cols = append(cols, uiGraphColumn{
+				Vals: []uiGraphValue{{Val: float32(hist.covered[date]) / float32(hist.instrumented[date])}},
```

Comment:
Nit: Do we want to protect against zero division just in case?

Reply:
If we want to see statistics about the compiled-in but excluded from the instrumentation files.... it probably makes sense.
It wasn't planned but the change is cheap. Let's just skip the files with 0 instrumentation points.

Reply:
Ah, got it. It was a sum on the day. Let's add it anyway.

Reply:
Done.